tools/actions/pr-open.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
```

Comment:
We don't need a JS file for it. All of it can be handled directly into the action.

Reply:
This is checking more than just the dependencies, that's just a feature I added so it's up-to-date with what the security working group plans